qa/supply-chain/audits.toml

```diff
@@ -0,0 +1,116 @@
+
+# cargo-vet audits file
+
+[criteria.crypto-reviewed]
+description = "The cryptographic code in this crate has been reviewed for correctness by a member of a designated set of cryptography experts within the project."
+
+[criteria.license-reviewed]
+description = "The license of this crate has been reviewed for compatibility with its usage in this repository. If the crate is not available under the MIT license, `contrib/debian/copyright` has been updated with a corresponding copyright notice for files under `depends/*/vendored-sources/CRATE_NAME`."
```

Comment:
I've added this as a possibly-useful criteria, but it probably has enough overlap with `cargo-deny` that the latter might be more useful here. See also https://github.com/mozilla/cargo-vet/issues/82#issuecomment-1149660622 where `cargo-deny` might end up using `cargo-vet` as a library to offer a superset of its functionality.